user_guide_src/source/outgoing/localization.rst

```diff
@@ -327,3 +327,29 @@ Detailed information can be found by running the command:
 .. code-block:: console
 
     php spark lang:find --help
+
+.. _sync-translations-command:
+
+Synchronization  Translation Files via Command
+==============================================
+
+.. versionadded:: 4.6.0
+
+After working on your translation for the current language for a long time, in some cases you will need to create files for another language.
+The ``spark lang:find`` command can be used, it is not prohibited. But it cannot fully detect absolutely all translations. Especially if the parameters are dynamically set as ``lang('App.status.' . $key, ['payload' => 'John'], 'en')``.
```

Comment:
I don't know why you added `, it is not prohibited`. What do you want to say by it?

Reply:
Ah, I don't get the following whole explanation.

> The spark lang:find command can be used, it is not prohibited. But it cannot fully detect absolutely all translations. Especially if the parameters are dynamically set as lang('App.status.' . $key, ['payload' => 'John'], 'en'). In this case, the best solution is to copy the finished language files and translate them. This way you can save your unique keys that the command did not find.

"to copy the finished language files and translate them"
What do  I need to do exactly? If I copy the file, there is no need of `lang` commands.

I also don't get why the explanation is there. What's the relation with the `lang:sync` command?

Reply:
I warned that my English is bad. 

I wrote that the `lang:find` helps you find NEW keys. What if you don't need to search? Or `lang:find` couldn't find complex translations as `lang('App.title', [], $locale)`? 
In this case, you must manually add the translation. 

> What do I need to do exactly? If I copy the file, there is no need of lang commands.

You will copy a identical array. After that, you need to manually search for and change existing keys (if available). When there is no file for `--target` locale, yes it is logical (copy-paste). 
Command is more suitable for merging.

Is it embarrassing? Then why do the `make:controller` and others commands, is it possible to copy an available controller?

Reply:
1. remove `, it is not prohibited`. I think it is not needed.
2. move the whole explanation to the section "Generating Translation Files via Command". Because it is an explanation for `lang:find`, not `lang:sync`.